src/pages/Analytics/Analytics.tsx

```diff
@@ -0,0 +1,16 @@
+import TopPools from '../../components/Global/Analytics/TopPools';
+
+export default function Analytics() {
+    return (
+        <section
```

Comment:
This would be something that we can extract as well, as its used by all the main pages in the app (no need to change for all the pages yet, but lets create a component that can be used by everyone when we do refactor)